### PR TITLE
#15 retrieve specific item

### PR DESCRIPTION
**Pull request by Muhet**
#### What does this PR do?
As a buyer shoold view a specific available item
As a vender or seller should view only his/her onw item.
#### Description of Task to be completed?
if a seller tried to view the item which is not his own he should get the message which says that he is not authorized
#### How should this be manually tested?
npm run dev for running server
in postman use http://localhost:3000/api/products/6 if you are seller with get given token to get the item
use GET method
#### Questions:

### CODE REVIEW

**Review by NTElissa — COMMENTED** [review]
good progress

**Review by faid-terence — APPROVED** [review]
Good Work 👏

**Review by ALVINdimpos — APPROVED** [review]

**Review by vanessa-abewe — APPROVED** [review]

**Review by yobuir — APPROVED** [review]

File: dist/app.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
+"use strict";
+
```
Comment (chanceDira) [inline dist/app.js]: why did you use /dist folder? I think it should be in gitignore other you can explain how you are using it

**Review by chanceDira — COMMENTED** [review]
Nice work @Muhet,

just one thing about dist folder can you clarify

**Review by chanceDira — APPROVED** [review]
Nice work @Muhet

**Pull request merged**